<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageViewerWindow</class>
 <widget class="QMainWindow" name="ImageViewerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageViewerWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="image">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pbIniciar">
        <property name="text">
         <string>Iniciar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbPausar">
        <property name="text">
         <string>Pausar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbParar">
        <property name="text">
         <string>Parar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbAutoInicio">
        <property name="text">
         <string>Auto-inicio</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbSalir">
        <property name="text">
         <string>Salir</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Archivo">
    <property name="title">
     <string>&amp;Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionCapturar"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuAy_uda">
    <property name="title">
     <string>Ay&amp;uda</string>
    </property>
    <addaction name="actionAcerca_de"/>
   </widget>
   <widget class="QMenu" name="menu_Editar">
    <property name="title">
     <string>&amp;Editar</string>
    </property>
    <addaction name="actionPreferencias"/>
   </widget>
   <addaction name="menu_Archivo"/>
   <addaction name="menu_Editar"/>
   <addaction name="menuAy_uda"/>
  </widget>
  <action name="actionAbrir">
   <property name="text">
    <string>A&amp;brir...</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>&amp;Salir</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca de...</string>
   </property>
  </action>
  <action name="actionCapturar">
   <property name="text">
    <string>Capturar</string>
   </property>
  </action>
  <action name="actionPreferencias">
   <property name="text">
    <string>Prefere&amp;ncias</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
